### ISSUE TITLE
ProgHackNight meta issue

### ISSUE THREAD

**Issue by schuyler1d** [ProgressiveCoders/Spoke#458]

**Comment by ChuckBTaylor** [comment]
Hello from from the HackNight

**Comment by nabchar** [comment]
🚀

**Comment by veekas** [comment]
👋

**Comment by zluo16** [comment]
Hey! This is Mike!

**Comment by JeremyParker** [comment]
Hey there!

**Comment by aardestar** [comment]
Hi everyone!

**Comment by schuyler1d** [comment]
Hi everyone!  We believe we have a fix for the auth0 issues.
Could some of you test out this branch:
https://github.com/MoveOnOrg/Spoke/pull/467
and see if you get past our Auth0 login step?

Thanks @hiemanshu for investigating!

**Comment by nabchar** [comment]
@schuyler1d awesome. I'll be back at proghacknight tonight and will plan on testing then. Happy to set up shop with whoever will be there!

**Comment by schuyler1d** [comment]
@ben-pr-p has volunteered to run the hacknight working group tonight.  He's made commits, was at the HackDay in December, and is running a production instance for ~~Brand New Congress~~ Justice Democrats.  I'll let him introduce himself, but I'm excited about our working group continuing and will see you all in the next hacknight.

(edit, updated Ben's org)

**Comment by zluo16** [comment]
Hey! Did anyone ever fix that user serialization issue? 

A couple of us got the auth0 working last week with the help of the auth0 fix (thanks for that) but we ran into an issue where passport was failing to serialize the logged in user into sessions. Did anyone end up resolving this issue or did anyone run into this issue in the past?

**Comment by schuyler1d** [comment]
@zluo16 and others: can you try @hiemanshu 's branch?
https://github.com/MoveOnOrg/Spoke/pull/467
(see comment thread and commits for evolution)

**Comment by zluo16** [comment]
@schuyler1d Thanks for that! It looks like the most recent change took care of the user serialization issue.

**Comment by nabchar** [comment]
That's great news. I'll see if I can take a look and try to finish getting set up locally.

**Comment by schuyler1d** [comment]
@nabchar please report back either here or on @hiemanshu 's PR.

**Comment by zluo16** [comment]
The latest update helped a lot with auth0 and user serialization. However, the wrap function in my app-renderer is still throwing up a 'Not found' error.

**Comment by JeremyParker** [comment]
The app is working for me locally on the `fix-auth0-login` branch. Thanks for all the help!

**Comment by lperson** [comment]
Hey!

**Comment by tommyyork** [comment]
Hi, tommy york!

**Comment by lperson** [comment]
👋

**Comment by gracecarey** [comment]
Hi!

**Comment by dtoakley** [comment]
hi folks!

**Comment by johntenezaca7** [comment]
Hey folks! It's John!

**Comment by mkc1** [comment]
hello! this is meredith

**Comment by eToThePiIPower** [comment]
Hey, this is Charles.

**Comment by 1kindjune** [comment]
Hi! This is June~ I'm doing this from prog hack night!

**Comment by rockycodes** [comment]
hi! i came to progressive hacknight yayyy.

**Comment by millmason** [comment]
Hi! This is Julia

**Comment by mehyar500** [comment]
Hey this is Mehyar.
Would love to get involved

**Comment by PhilNYC** [comment]
Howdy, interested in helping out.

**Comment by DAlperin** [comment]
Hello! Interested in helping out

**Comment by lperson** [comment]
👋

**Comment by lukeburns** [comment]
Hello! Interested in working on the profanity filter! @schuyler1d

**Comment by lperson** [comment]
@lukeburns: sign up [here](https://airtable.com/shraBEItZa0sYiMB0) to get on the slack

**Comment by lperson** [comment]
@lukeburns check your email -- you must do a progcode onboarding before they'll invite you to the slack.

**Comment by schuyler1d** [comment]
@DAlperin I heard you were interested in easy heroku installation of spoke.  Besides improving defaults, etc, There's a project: https://github.com/ToolsForRadicals/spoke-deploy
It has a slack channel `#spoke-deploy` on http://progco.de/join

**Comment by rcackerman** [comment]
👊

**Comment by yeti-detective** [comment]
Hello! I'm Chris Brown

**Comment by mjohnson324** [comment]
Hi I'm Michael, here at Progressive Hack Night! Trying to get started helping with Spoke

**Comment by michaelwolfnyc** [comment]
Hey, I'm Michael at Progressive Hack Night learning about Spoke.

**Comment by harpojaeger** [comment]
Hi Michael! Welcome!

**Comment by emestabillo** [comment]
Hi everyone! Had a great time at Progressive Hack Night!

**Comment by arena** [comment]
Hello Hackers!!!! 
request/suggestion for upcoming hack night: dev environment troubleshooting for anyone who needs it.

**Comment by Frydafly** [comment]
@ibrand is this issue/thread still in use/ useful?

**Comment by Arique1104** [comment]
Hello,

My name is Arique (they/them) and I am the Community Manager for Spoke. First of all, thank you so much for creating this issue and for everyone’s contributions to this ticket.

I am going through all of our issues and beginning to close all issues created in 2018 or older. Your issue qualifies.

I am giving everyone one week to answer the following question: 

- **Has this issue been resolved during the subsequent Spoke Updates?**

On November 16th I will be closing this issue if:
1. There is no response
2. You respond yes

If you are reading this message after November 16th, please note that if you tag me and answer the above question, I will be happy to reinstate the issue.

**Comment by ibrand** [comment]
I'll provide a little context on this for our own historical record, but yes it can (and should be) closed.

We used meta issues on the Spoke project as one of the places to gather volunteers from a specific event or group. People who came into the project from Progressive HackNight (an event that no longer exists) would comment on this meta issue to either "check in" so that we had their github handle as a contact point and we could later more easily add them to slacks or would specifically mention an issue they wanted to work on that we could then assign to them.

This was a pretty chaotic and inconsistent intake method but also worked really well, and notified other people on the issue who had their notifications on for the thread when new people would join. There are even moments when folks would say hi to each other on threads like this. We also tried this with Grace Hopper on issue #1230 but we didn't have as strong of a prior relationship with that community.

With Progressive HackNight no longer running and us using different volunteer management strategies, I'm closing this.
